biogeophys/EDAccumulateFluxesMod.F90

```diff
@@ -82,15 +82,13 @@ subroutine AccumulateFluxes_ED(nsites, sites, bc_in, bc_out, dt_time)
 
                    if ( debug ) then
```

Comment:
we should remove this debug statement.  we can add these types of write statements in when needed, but not maintained in the main codebase